markdown.module

```diff
@@ -103,8 +103,8 @@ And now some code:
  */
 function _filter_markdown($text, $format) {
   if (!empty($text)) {
-    module_load_include('php', 'markdown', 'markdown');
-    $text = Markdown($text);
+    module_load_include('inc.php', 'markdown', 'includes/MarkdownExtra');
```

Comment:
I don't think switching from Markdown to MarkdownExtra is a trivial change, that's why I made the Markdown Extra module: https://github.com/backdrop-contrib/markdownextra